api/hasura/actions/_handlers/updateCircle.ts

```diff
@@ -2,6 +2,8 @@ import { VercelRequest, VercelResponse } from '@vercel/node';
 
 import { authCircleAdminMiddleware } from '../../../../api-lib/circleAdmin';
 import { endNominees, updateCircle } from '../../../../api-lib/gql/mutations';
+import * as queries from '../../../../api-lib/gql/queries';
```

Comment:
can we just import getCircle and not the entire `*` ?

Reply:
Refactored with https://github.com/coordinape/coordinape/pull/1360/commits/3793e52a6ff515e4f93eb86940ff8e2763148fd6